### PR TITLE
Edit README.md and the CHANGESLOG.md with the new commands and how to install

### PR DESCRIPTION
**Pull request by Bishoy-at-pieces**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,45 @@ Need the copilot to comment, fix a bug, or add a doc string? Simply go to the co
...
 
-### Connect with the Pieces Community
+### Connect/Disconnect your cloud
+
+You can connect and disconnect from your cloud using  `Pieces: Connect your cloud` or `Pieces: Disconnect your cloud` command in your command palette.(Login required!)
+
+### Search
+
+You can search a saved material using `Pieces: Search`.
+
```
Comment (mark-at-pieces) [inline README.md:46]: TODO go deeper on this in the future, ie w/ different search modes, this will explicitly only search your Snippets/Assets

**Review by mark-at-pieces — APPROVED** [review]
didnt go super deep on the copy but looking good

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,10 @@
 # CHANGELOG
+# 1.0.0
+## May 22th, 2024
+
+- Add support command
```
Comment (mack-at-pieces) [inline CHANGELOG.md:5]: Can you pls teach the user how they'd use the new command and what it's for in this changelog?

i.e. "Added a support command, "Pieces: Get Support", to access our public support GitHub repository or connect with the Pieces team."

File: README.md [hunk 3]
```diff
@@ -36,7 +36,45 @@ Need the copilot to comment, fix a bug, or add a doc string? Simply go to the co
 
 You can log in and out of your Pieces account using the `Pieces: Login` or `Pieces: Logout` command in your command palette. An output panel is available to check your cloud status and login status.
 
-### Connect with the Pieces Community
+### Connect/Disconnect your cloud
+
+You can connect and disconnect from your cloud using  `Pieces: Connect your cloud` or `Pieces: Disconnect your cloud` command in your command palette.(Login required!)
```
Comment (mack-at-pieces) [inline README.md:41]: Can these commands be "Pieces: Disconnect Personal Cloud" and "Pieces: Connect Personal Cloud" ?

File: README.md [hunk 4]
```diff
@@ -36,7 +36,45 @@ Need the copilot to comment, fix a bug, or add a doc string? Simply go to the co
...
+
+### Search
+
+You can search a saved material using `Pieces: Search`.
+
+### Reload
+
+Run `Pieces: Reload the plugin` if you can't see any command in your command palette. Just make sure that **Pieces OS** is running
```
Comment (mack-at-pieces) [inline README.md:49]: Can this command be "Pieces: Reload Plugin" ?

File: README.md [hunk 5]
```diff
@@ -36,7 +36,45 @@ Need the copilot to comment, fix a bug, or add a doc string? Simply go to the co
...
+
+### Support 
+
+Got stuck? No worries, Simply go to the command palette and run `Pieces: Get Support`. you will find some very awsome resources where you can connect with use
```
Comment (mack-at-pieces) [inline README.md:53]: "Experiencing an issue or have feedback for the Pieces team? No worries, we've got you covered. Simply open the command palette and run `Pieces: Get Support`. From here, you will find resources where you can connect with the Pieces team."

File: README.md [hunk 6]
```diff
@@ -36,7 +36,45 @@ Need the copilot to comment, fix a bug, or add a doc string? Simply go to the co
...
+
+### Search
+
+You can search a saved material using `Pieces: Search`.
+
+### Reload
+
+Run `Pieces: Reload the plugin` if you can't see any command in your command palette. Just make sure that **Pieces OS** is running
+
+### Support
```
Comment (mack-at-pieces) [inline README.md:51]: "Get Support or Share Feedback"

File: README.md [hunk 7]
```diff
@@ -36,7 +36,45 @@ Need the copilot to comment, fix a bug, or add a doc string? Simply go to the co
...
-### Connect with the Pieces Community
+### Connect/Disconnect your cloud
+
+You can connect and disconnect from your cloud using  `Pieces: Connect your cloud` or `Pieces: Disconnect your cloud` command in your command palette.(Login required!)
+
+### Search
+
+You can search a saved material using `Pieces: Search`.
+
+### Reload
```
Comment (mack-at-pieces) [inline README.md:47]: "Reload the Plugin"

File: README.md [hunk 8]
```diff
@@ -36,7 +36,45 @@ Need the copilot to comment, fix a bug, or add a doc string? Simply go to the co
 
 You can log in and out of your Pieces account using the `Pieces: Login` or `Pieces: Logout` command in your command palette. An output panel is available to check your cloud status and login status.
 
-### Connect with the Pieces Community
+### Connect/Disconnect your cloud
+
+You can connect and disconnect from your cloud using  `Pieces: Connect your cloud` or `Pieces: Disconnect your cloud` command in your command palette.(Login required!)
+
+### Search
```
Comment (mack-at-pieces) [inline README.md:43]: "Find your Saved Materials"

File: README.md [hunk 9]
```diff
@@ -36,7 +36,45 @@ Need the copilot to comment, fix a bug, or add a doc string? Simply go to the co
...
+### Support 
+
+Got stuck? No worries, Simply go to the command palette and run `Pieces: Get Support`. you will find some very awsome resources where you can connect with use
+You can choose the best suited for you and the link will be automatically opened in the browser.
+
+## How to install?
+
+### Stable build (Recommended):
+
+- Download the [package control](https://packagecontrol.io/installation)
```
Comment (mack-at-pieces) [inline README.md:60]: Should this be: "Download the package from [package control](https://packagecontrol.io/installation)"

File: README.md [hunk 10]
```diff
@@ -36,7 +36,45 @@ Need the copilot to comment, fix a bug, or add a doc string? Simply go to the co
...
+- Open your command palette and search for `Package Control: Install Package`
```
Comment (mack-at-pieces) [inline README.md:61]: Is there a hotkey for opening the command palette?

File: README.md [hunk 11]
```diff
@@ -36,7 +36,45 @@ Need the copilot to comment, fix a bug, or add a doc string? Simply go to the co
...
+Got stuck? No worries, Simply go to the command palette and run `Pieces: Get Support`. you will find some very awsome resources where you can connect with use
+You can choose the best suited for you and the link will be automatically opened in the browser.
+
+## How to install?
+
+### Stable build (Recommended):
+
+- Download the [package control](https://packagecontrol.io/installation)
+- Open your command palette and search for `Package Control: Install Package`  
+- Seach for Pieces and Choose it
```
Comment (mack-at-pieces) [inline README.md:62]: Typo here: "Search for Pieces and select it"

**Review by mack-at-pieces — CHANGES_REQUESTED** [review]

File: README.md [hunk 12]
```diff
@@ -36,7 +36,45 @@ Need the copilot to comment, fix a bug, or add a doc string? Simply go to the co
...
+- **IMPORTANT: When you first download the package it will download some dependencies, so you might need to restart sublime after downloading**
+
+### Pre-releases:
```
[the remark this answers is not in the document]
Comment (Bishoy-at-pieces) [reply README.md:65]: Nathan will finish the github action and there should be a pre-release every push to main

File: README.md (hunk 9 above)
Comment (Bishoy-at-pieces) [reply README.md:60]: Nope you need to download the package control first it is not downloaded by default by sublime

File: README.md (hunk 1 above)
Comment (Bishoy-at-pieces) [reply README.md:46]: so yes as long as you entering some charaters in the query the results will be shown when you hit enter a list will be available to choose an asset from to open it

Comment (Bishoy-at-pieces) [reply README.md:46]: so yes as long as you entering some characters in the query the results will be shown when you hit enter a list will be available to choose an asset from to open it

Comment (Bishoy-at-pieces) [reply README.md:46]: so yes as long as you entering some charaters in the query the results will be shown when you hit enter a list will be available to choose an asset from to open it

File: README.md [hunk 13]
```diff
@@ -30,13 +30,61 @@ To save a new snippet in Pieces, select the code you wish to save, right-click t
...
+Need the copilot to comment, fix a bug, or add a doc string?Select the text right-click to open the context menu, and select `Pieces > Ask Copilot`.
 
 ### Login/Logout
 
 You can log in and out of your Pieces account using the `Pieces: Login` or `Pieces: Logout` command in your command palette. An output panel is available to check your cloud status and login status.
 
-### Connect with the Pieces Community
+### Connect or Disconnect your Personal Pieces Cloud
+
+You can connect and disconnect from your cloud using  `Pieces: Connect Personal Cloud` or `Pieces: Disconnect Personal Cloud` command in your command palette.
```
Comment (mack-at-pieces) [inline README.md:41]: There's an extra space between "using" and "`Pieces:...`"

File: README.md [hunk 14]
```diff
@@ -30,13 +30,61 @@ To save a new snippet in Pieces, select the code you wish to save, right-click t
...
+### Get Support or Share Feedback
+
+Experiencing an issue or have feedback for the Pieces team? No worries, we've got you covered. Simply open the command palette and run `Pieces: Get Support`. From here, you will find resources where you can connect with the Pieces team.
+
+## How to install?
+
+### Stable build (Recommended):
+
+- Download the [package control](https://packagecontrol.io/installation)
+- Open your command palette. You can use the shortcut key combination `Ctrl+Shift+P` on Windows and `Cmd+Shift+P` on OSX.
```
Comment (mack-at-pieces) [inline README.md:69]: Lets use "MacOS" here for consistent terminology

**Review by mack-at-pieces — CHANGES_REQUESTED** [review]

**Review by mack-at-pieces — APPROVED** [review]

**Pull request merged**